### ISSUE TITLE
Errors on Javascript console, steady spinner for a virtual server homepage

### ISSUE THREAD

**Issue by 7starsone** [qooob/authentic-theme#311]
```The select2('close') method was called on an element that is not using Select2.
h.fn.select2 @ package.min.js:26 ```
AND
```Uncaught TypeError: Cannot read property 'close' of undefined
package.min.js:26 ```

First time it happens... it freezes and you should reload the page
![Image of steady spinner](http://i63.tinypic.com/2ntle0n.jpg)

**Comment by qooob** [comment]
Yeah, I know about this. It's in my list of hunted bugs. However, it didn't require full page reload.

**Comment by 7starsone** [comment]
indeed at the moment it doesn't work even on page reload...

**Comment by qooob** [comment]
..not possible! :D

**Comment by 7starsone** [comment]
well, the first time... it did it again, reloading a second time and choosing that virtual server and worked... anyway, glad to know that you're aware of...
;-) best regards

**Comment by qooob** [comment]
I have distant idea of why it's happening.. Will try to fix it!

**Comment by qooob** [comment]
It will be fixed in 17.21! Thanks for pointing out!
